scale/job/models.py

```diff
@@ -2796,6 +2869,37 @@ def get_details_v5(self, job_type_id):
 
         return job_type
 
+    def get_details_v6(self, name, version):
+        """Returns the job type for the given name and version with all detail fields included.
+
+        The additional fields include: errors, job_counts_6h, job_counts_12h, and job_counts_24h.
+
+        :param name: The name of the job type.
+        :type name: string
+        :param version: The version of the job type.
+        :type version: string
+        :returns: The job type with all detail fields included.
+        :rtype: :class:`job.models.JobType`
+        """
+
+        # Attempt to get the job type
+        job_type = JobType.objects.all().get(name=name, version=version)
+
+        # Scrub configuration for secrets
+        if job_type.configuration:
+            if JobInterfaceSunset.is_seed_dict(job_type.manifest):
+                configuration = job_type.get_job_configuration()
+                manifest = SeedManifest(job_type.manifest, do_validate=False)
+                configuration.remove_secret_settings(manifest)
+                job_type.configuration = convert_config_to_v6_json(configuration).get_dict()
```

Comment:
Don't do a Seed vs legacy split here. The v6 details API should only return version v6 of any JSON schemas.
In the detail serializer, use the JobType.get_v6_configuration_json() function as the source (like how the BatchDetailsSerializerV6 uses `configuration = serializers.JSONField(source='get_v6_configuration_json')` for its configuration).